src/olympia/reviewers/forms.py

```diff
@@ -513,8 +514,21 @@ def __init__(self, *args, **kwargs):
 class WhiteboardForm(forms.ModelForm):
 
     class Meta:
-        model = Addon
-        fields = ['whiteboard']
+        model = Whiteboard
+        fields = ['private', 'public']
+        labels = {
+            'private': _("Private Whiteboard"),
```

Comment:
We're heavy users of single quotes, let's change that here please?

Reply:
Ah yes, surprised the linter didn't catch this.